onnx/defs/math/defs.cc

```diff
@@ -338,6 +373,26 @@ tensor elementwise.
     .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
         "Constrain input and output types to float tensors.");
 
+OPERATOR_SCHEMA(HardSigmoid)
+  .NumInputs(1)
+  .NumOutputs(1)
+  .AllowConsumed({{0, 0}})
+  .Attr("alpha",
+        "Value of alpha",
+        AttrType::FLOAT)
+  .Attr("beta",
+        "Value of beta",
+        AttrType::FLOAT)
+  .SetDoc(R"DOC(
+HardSigmoid takes one input data (Tensor<T>) and produces one output data
+(Tensor<T>) where the HardSigmoid function, y = max(0, min(1, alpha * x + beta)),
```

Comment:
for ops like this would be cool to provide "lowering" pass that can translate into more elemental ops if necessary (min/max/arithmetics in this case). But probably not for now

Reply:
Right. Probably not for now.

Reply:
Right. Probably not for now